config/develop/synapse-login-scipooldev-params.yaml

```diff
@@ -33,3 +33,5 @@ sceptre_user_data:
           {"teamId":"3409012","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"},
           {"teamId":"3409013","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogExternalEndusers"}
         ]'
+    - Name: MarketplaceProductCodeSC
+      Value: d3xz0ef8n2bqgnuynummc21w7
```

Comment:
please remove.  the ImportValue (in templates/app.yaml) for MARKETPLACE_ID_DYNAMO_TABLE_NAME already does the trick.

Reply:
Sorry, I don't follow:  This is MarketplaceProductCodeSC, not MARKETPLACE_ID_DYNAMO_TABLE_NAME.

Reply:
ohh, my mistake.  this looks good.